src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1

```diff
@@ -14,6 +14,51 @@
 
 <#
 .SYNOPSIS
+Virtual network express route gateway connection tests
+#>
+function Test-VirtualNetworkeExpressRouteGatewayConnectionCRUD
+{
+    # Setup
+    $rgname = "onesdkTestConnection"
+    $vnetConnectionName = Get-ResourceName
+	$location = "westus"
+    try 
+     {
+      # Get the resource group
+      $resourceGroup = Get-AzureRmResourceGroup -Name $rgname  
+      Assert-NotNull $resourceGroup
+      # Get Gateway
+	  $gw = Get-AzureRmVirtualNetworkGateway -ResourceGroupName $rgname
+	  Assert-AreEqual 1 @($gw).Count
+      # Get Circuit
+      $circuit = Get-AzureRmExpressRouteCircuit -ResourceGroupName $rgname
+      Assert-AreEqual 1 @($circuit).Count
+	
+	  # Create & Get VirtualNetworkGatewayConnection
+      $actual = New-AzureRmVirtualNetworkGatewayConnection -ResourceGroupName $rgname -name $vnetConnectionName -location $location -VirtualNetworkGateway1 $gw  -ConnectionType ExpressRoute -RoutingWeight 3 -PeerId $circuit.Id
```

Comment:
@dihan0604 please change the indentation in this file (replace Tabs to 4 spaces)